### ISSUE TITLE
Determine how we leverage the document-me label in Documentation Project

### ISSUE THREAD

**Issue by jwrober** [longturn/freeciv21#496]
The label is to help with release notes. Thinking that we could have a tagged release notes page(s) in the docs.

**Comment by lmoureaux** [comment]
So what I did for the release notes of `alpha1` was to look at the list of merged PRs flagged `document-me` and build the list from there. I don't think this can be automated without imposing restrictions on eg the PR title.

**Comment by lmoureaux** [comment]
More specifically, I used [this request](https://github.com/longturn/freeciv21/issues?q=label%3Adocument-me+is%3Aclosed)

**Comment by jwrober** [comment]
So you are using the label as a "flag" of some kind to make it easier to generate the list of changes for a release?  If so, then what is the criteria for adding the label? What is the process to create a release?  This is probably a great use of a "Release Process" project in fc21 repo with a collection of notes in the ToDo board. Alternately we document in `docs` or a WiKi page?

**Comment by lmoureaux** [comment]
> So you are using the label as a "flag" of some kind to make it easier to generate the list of changes for a release?

Yes

>  If so, then what is the criteria for adding the label?

The change must possibly affect users. Ideally such changes would come with the relevant documentation updates (in the same PR), but since documentation is rather sparse it's not a realistic requirement.

> This is probably a great use of a "Release Process" project in fc21 repo with a collection of notes in the ToDo board. Alternately we document in docs or a WiKi page?

There are two things here. One is coming up with a release process (separate project) and the other is how we document it (related to the Doc project). The part in the Documentation Project is eventually related to the scope of the documentation that we want in `docs`...

**Comment by jwrober** [comment]
@lmoureaux and I were chatting on Discord. I think we are going to do draft tags and will document the draft as PRs are merged.

**Comment by lmoureaux** [comment]
Does this mean that `document-me` will no longer be used or shall we keep using it as a reminder?

**Comment by jwrober** [comment]
I do not think we should use the tag any longer. All PRs included in a tag will be added as we go along.
